src/libsyntax_pos/symbol.rs

```diff
@@ -150,9 +150,9 @@ newtype_index! {
 // The interner is pointed to by a thread local value which is only set on the main thread
 // with parallelization is disabled. So we don't allow `Symbol` to transfer between threads
 // to avoid panics and other errors, even though it would be memory safe to do so.
-#[cfg(not(parallel_queries))]
```

Comment:
CFGs like these seem very prone to invisible breakage (if we end up not running either configuration on CI). Perhaps we should maintain parallel-able code in all configurations?

Reply:
We do check both on PRs on Travis at least.